shelley/chain-and-ledger/formal-spec/address.tex

```diff
@@ -3,10 +3,18 @@ \section{Addresses}
 
 Addresses are described in section 4.2 of the delegation design document \cite{delegation_design}.
 The types needed for the addresses are defined in Figure~\ref{fig:defs:addresses}.
+They all involve a credential, which is either a key or a multi-signature script.
 There are four types of UTxO addresses:
 \begin{itemize}
 \item Base addresses, $\AddrB$, containing the hash of a payment credential and
-  the hash of a staking credential,
+  the hash of a staking credential. Note that the payment credential hash is the
+  hash of the key (or script) which has contol of the funds at this address,
+  i.e. is able to witness spending
+  them. The staking credential is the credential which accumulates
+  the rewards associated with the funds at the payment part of the address.
```

Comment:
Is it maybe confusing here to conflate the stake key and the reward address? Maybe we could say that the staking credential controls the delegation decision?

Reply:
Sounds good